app/templates/macros/answers.html

```diff
@@ -0,0 +1,66 @@
+{% macro textbox(value='') -%}
```

Comment:
Undecided as yet. You _could_ argue that it's each app's responsibility to decide what the interface is—think of the buyer app which displays the description of a service differently to the admin/supplier apps.